src/ImageRunModal.jsx

```diff
@@ -885,14 +883,14 @@ export class ImageRunModal extends React.Component {
                                 </Flex>
                             </FormGroup>
                         }
-                        {((this.props.userLingeringEnabled && this.props.userPodmanRestartAvailable) || (this.isSystem() && this.props.podmanRestartAvailable)) &&
+                        {((userLingeringEnabled && userPodmanRestartAvailable) || (this.isSystem() && podmanRestartAvailable)) &&
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1249-20230622-075526-286d4760-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)